C#/C#.sublime-syntax

```diff
@@ -227,6 +252,11 @@ contexts:
       scope: invalid.illegal.cs
 
   main:
+    - meta_include_prototype: false
+    - match: ''
+      push: [statements, shebang]
```

Comment:
With a dedicated `statements` context, can `main` be moved to the beginning next to `prototype`?

Seeing the main entry positioned in the middle of the file feels a bit strange.

Reply:
yes, done :)